.github/workflows/test.yml

```diff
@@ -20,6 +20,14 @@ jobs:
     - name: Build
       run: GOWORK=off CGO_ENABLED=0 go build -v
 
+    - name: staticcheck
```

Comment:
Can we enable staticheck for all go projects (bigquery-streaming-write-loader, engine proxy, ...)

Reply:
it shall arrive one day